test/token-pool-proxy.test.js

```diff
@@ -52,12 +43,12 @@ contract('Token Pool', accounts => {
       ctx.actors.owner,
       ctx.contracts.storage.address
     ])
-    ctx.contracts.reputation = await app.createProxy(ReputationManager, "ReputationManager", 'initialize', [
+    ctx.contracts.reputation = await app.createProxy(ReputationManager, 'ReputationManager', 'initialize', [
```

Comment:
there are helper functions in `init.js` you can use to instantiate all these contracts

Reply:
Leaving until end of updates as this is least critical